### PR TITLE
Solucion ejercicio 88 en js

### PR DESCRIPTION
**Pull request by alnavarroh**
Approve pls

### CODE REVIEW

File: algorithm_exercises/88. Merge Sorted Array/solution.js [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+            nums1[k] = nums1[m]
+            m--
+        }
+        else {
+            nums1[k] = nums2[n]
+            n--
+        }
+        k--
+    }
+};
```
Comment (Darova93) [inline algorithm_exercises/88. Merge Sorted Array/solution.js]: Add an empty line at the end please.
Can you rename the variables so the algorithm is easier to read

**Review by Darova93 — CHANGES_REQUESTED** [review]

File: algorithm_exercises/88. Merge Sorted Array/solution.js (hunk 1 above)
Comment (alnavarroh) [reply algorithm_exercises/88. Merge Sorted Array/solution.js]: addressed in new commit

**Review by Eduardo-93 — APPROVED** [review]
lgtm

**Review by Bydel93 — APPROVED** [review]
lgtm

**Review by Darova93 — APPROVED** [review]

**Pull request merged**